### ISSUE TITLE
Include Translation

### ISSUE THREAD

**Issue by snakyjake1** [DanteCoder/JoplinBibleQuote#18]
Might it be possible to include the translation?

Current:
Genesis 1:1
In the beginning God created the heavens and the earth.

Proposed:
Genesis 1:1 (NIV)
In the beginning God created the heavens and the earth.

**Comment by DanteCoder** [comment]
Yeah, it would be a good addition. I'll consider it in the next update.
